### DISCUSSION TITLE
Is there a way to trigger sync of the page when the original language version is edited?

### DISCUSSION THREAD

**Discussion by Algoinde** [wagtail/wagtail-localize#570] [Q&A] [answered]
I don't want the editors to need to menu around to manually sync the translations on all pages they edit. Can this be done automatically on page edit?

**Comment by chris48s** [comment, marked as answer]
It doesn't exist in core or localize out of the box, but you could implement what you're suggesting with a small hook. I haven't tested this snippet, but I think this would do what you're suggesting:

```py
from wagtail.core import hooks
from wagtail_localize.models import TranslationSource

@hooks.register('after_edit_page')
def after_edit_page(request, page):
    source = TranslationSource.objects.get_for_instance(page)
    source.update_from_db()
```

or you might consider doing it on `after_publish_page`.
